src/screens/NewsDetailsScreen.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import { View, StyleSheet } from 'react-native';
+import { NewsDetailsBody } from '../components';
+
+export default class NewsDetailsScreen extends Component {
+  static navigationOptions = {
+    title: 'Details',
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.handleDetailsClose = this.handleDetailsClose.bind(this);
+    this.handleOpenWebArticle = this.handleOpenWebArticle.bind(this);
+
+    const { article } = props.navigation.state.params;
+    this.state = { article };
+  }
+
+  handleDetailsClose() {
+    const { onDetailsClose } = this.props;
+    onDetailsClose();
+  }
+
+  handleOpenWebArticle() {
+    const { navigation } = this.props;
+    const { article } = this.state;
```

Comment:
`const { article: { web_url }} = this.state;`

but if you expect that `article` can be `null`, pls use `_.get(article, 'web_url')`